lib/mastodon/search_cli.rb

```diff
@@ -15,7 +15,7 @@ class SearchCLI < Thor
       changed since the last run.
     LONG_DESC
     def deploy
-      processed = Chewy::RakeHelper.upgrade
+      processed = Chewy::RakeHelper.upgrade parallel: true
       Chewy::RakeHelper.sync(except: processed)
```

Comment:
This method also accepts `parallel` argument

Reply:
I agree with @Kjwon15 that it would make sense to expose the `p` option for the command and then pass it as `parallel` to both of these methods

Reply:
We could default it to 2, and make sure `parallel: false` when `-p=1` (or 0)

Reply:
How about this?

With the --parallel option, Execute tasks in parallel. The default is 2, which specifies the number of processors. If true is specified, it is automatically derived.

Reply:
'auto' may be more appropriate than 'true'.

Reply:
Agree

Reply:
`sync` is probably the heaviest part of the task, it also needs `parallel` argument